### PR TITLE
[MOBILE-1280] Add number attribute support to Cordova

### PR DESCRIPTION
**Pull request by BrianBatchelder**
### What do these changes do?
Add number channel attribute support to Cordova plugin for Android and iOS.

### Why are these changes necessary?
Support the attributes.

### How did you verify these changes?
Manually set through javascript. Checked result using cool new API endpoint I got from Weasel.

#### Verification Screenshots:
image

### CODE REVIEW

File: www/UrbanAirship.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ function AttributesEditor(nativeMethod) {
    * @return {AttributesEditor} The attribute editor instance.
    */
   editor.setAttribute = function(name, value) {
-    argscheck.checkArgs('ss', "AttributesEditor#setAttribute", arguments)
+    argscheck.checkArgs('s*', "AttributesEditor#setAttribute", arguments)
```
Comment (marc-scig) [inline www/UrbanAirship.js]: asterisk means "anything goes", right? is that what we want?

File: src/android/UAirshipPlugin.java [hunk 2]
```diff
@@ -1067,10 +1067,17 @@ void editChannelAttributes(@NonNull JSONArray data, @NonNull CallbackContext cal
 
             String action = operation.optString(ATTRIBUTE_OPERATION_TYPE);
             String key = operation.optString(ATTRIBUTE_OPERATION_KEY);
-            String value = operation.optString(ATTRIBUTE_OPERATION_VALUE);
 
             if (ATTRIBUTE_OPERATION_SET.equals(action)) {
-                editor.setAttribute(key, value);
+                Object value = operation.opt(ATTRIBUTE_OPERATION_VALUE);
+
+                if (value == JSONObject.NULL) {
```
Comment (marc-scig) [inline src/android/UAirshipPlugin.java]: Do we want to compare with actual null, here? What happens if someone passes null or undefined on the JS side to an unchecked argument?

https://developer.android.com/reference/org/json/JSONObject#opt(java.lang.String)

File: www/UrbanAirship.js (hunk 1 above)
Comment (marc-scig) [reply www/UrbanAirship.js]: (I assume there's a good reason for it but I thought I'd ask)

Comment (BrianBatchelder) [reply www/UrbanAirship.js]: It does mean "any type". I'd prefer to indicate "number or string", but `checkArgs()` doesn't support that.

File: src/android/UAirshipPlugin.java (hunk 2 above)
Comment (BrianBatchelder) [reply src/android/UAirshipPlugin.java]: That's how I tested this, by passing in `null`. It gets assigned to `value` as `JSONObject.NULL`. The javascript I tested was:

`UAirship.editChannelAttributes().setAttribute("teststring","testy").setAttribute("testnull",null).setAttribute("testnumber",3).apply()`

In this case we skipped setting the "testnull" attribute, which seems like the correct behavior.

Comment (marc-scig) [reply src/android/UAirshipPlugin.java]: What about undefined?

Comment (BrianBatchelder) [reply src/android/UAirshipPlugin.java]: Tested with `undefined`. It also works fine. We skip setting any attribute with value `undefined`.

**Pull request merged**